.gitignore

```diff
@@ -180,3 +180,6 @@ project.lock.json
 .vs/
 Tests/
 TestFiles/
+DocumentFormat.OpenXml.Tests/Spreadsheet.xltx
```

Comment:
Ok - I've create #166 to track cleaning up the test infrastructure. For now, we probably don't need this change (it'll be a good impetus to fix that issue)